test/PooltogetherGovernor.t.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.18;
+
+import {PooltogetherGovernorTest} from "test/helpers/PooltogetherGovernorTest.sol";
+
+contract Constructor is PooltogetherGovernorTest {
+  function testFuzz_CorrectlySetsAllConstructorArgs(uint256 _blockNumber) public {
+    assertEq(governorBravo.name(), "Pooltogether Governor Bravo");
+    assertEq(address(governorBravo.token()), POOL_TOKEN);
+    // These values were all copied directly from the mainnet alpha governor at:
+    // 0xB3a87172F555ae2a2AB79Be60B336D2F7D0187f0
+    assertEq(INITIAL_VOTING_DELAY, 1);
```

Comment:
We have this asserts in the GC codebase but seeing them here I don't see what good they do. I think we can remove them for the other checks described in other comments below.

Reply:
https://github.com/ScopeLift/pooltogether-governor-upgrade/pull/9/commits/2f07ad167101bbab14f62020236c56e22f6c1b3c